soa-suite/README.md

```diff
@@ -0,0 +1,320 @@
+# Deploy Oracle SOA Suite on Kubernetes with the WebLogic Operator
+
+The quick and easy way to deploy this chart on Oracle Kubernetes Engine (OKE) on Oracle Cloud Infrastructure (OCI) is to go check out the related Terraform template here [fixme](fixme), or the step by step guide 
+
+The Terraform template helps deploy some of the Kubernetes dependencies, however if you are deploying on other cloud providers, you will need to provision the persistent storage (as well as the database, for a production deployment) 
+
+To deploy manually, review the instructions below: 
+
+## Disclaimer
+
+This deployment of Oracle SOA Suite makes use of the Oracle SOA Suite Helm Chart based on the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release.
+
+The Helm chart is provided as an example and is not currently officially supported by Oracle. Refer to the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release for the officially supported deployment.
+
+## Caveats
+
+Although this release follows the same flow as the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release, only the Traefik ingress controller is currently supported.
+
+
+## 1. Mandatory Prerequisites
+
+To deploy this chart, you will need:
+
+- An Oracle Container Registry account
+- Acknowledge terms of use for the Oracle SOA Suite Docker image on the Oracle Container Registry.
+- Acknowledge terms of use for the Oracle Database Docker image on the Oracle Container Registry to deploy the test Oracle database in your cluster (NOT for production).
+- A Network File Server (NFS) and mount point accessible from within your cluster.
+
+### 1.1 Oracle Container Registry
+
+You will need an Oracle Container Registry account here: [https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::](https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::)
```

Comment:
Oracle Container Registry uses Oracle Single Sign-On, so we should probably point folks at https://profile.oracle.com/myprofile/account/create-account.jspx

Reply:
done